mappings/net/minecraft/block/entity/BlockEntity.mapping

```diff
@@ -25,7 +25,7 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
-	METHOD method_11009 setWorld (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
+	METHOD method_11009 setLocation (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
```

Comment:
is this more like init location (i.e. only called once in lifetime or after revalidation)

Reply:
It's called each time `setBlockEntity` is called, so I think `set` is better than `init` in this case

Reply:
Isn't `setBlockEntity` only called once per BE?

Reply:
Nope it's not. Additionally, `setLocation` is called in `loadBlockEntity`